### PR TITLE
Binary Search Tree and traversal in python

### PR DESCRIPTION
**Pull request by Blesson2598**
Fixes #170

### CODE REVIEW

**Comment by jenistenxavier** [comment]
@Blesson2598 : It would be great if you can add the description along with the issue number.
Feel free to use our contribution guidelines (https://github.com/fnplus/interview-techdev-guide/blob/master/CONTRIBUTING.md) for reference

**Comment by jenistenxavier** [comment]
You have to link the PR to an issue. Check this out 
https://github.com/fnplus/interview-techdev-guide/blob/master/CONTRIBUTING.md#how-to-contribute-an-implementation-code

File: Algorithms/Tree Traversal/Binary Search Tree/BST.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    if root:
+        print(root.val,end=" ")
+        preorder(root.left)
+        preorder(root.right)
+def postorder(root):
+    if root:
+        postorder(root.left)
+        postorder(root.right)
+        print(root.val,end=" ")
+arr=input()
```
Comment (Blesson2598) [inline Algorithms/Tree Traversal/Binary Search Tree/BST.py]: Get the input from the user

**Comment by xlogix** [comment]
Thanks for contributing!

**Pull request merged**